python3/bin/perfmon

```diff
@@ -112,6 +112,9 @@ class UsageException(Exception):
     pass
 
 
+class NotGetValueException(Exception):
```

Comment:
I think this is equal to the `bare-except`.
From the comments, I think the code logics expect some Exceptions like `IndexError` or `KeyError`.